images/operator-registry/Dockerfile.registry.ci

```diff
@@ -1,19 +1,19 @@
-FROM quay.io/operator-framework/upstream-registry-builder:v1.13.9 as registry-builder
+FROM quay.io/operator-framework/upstream-registry-builder:v1.30.1 as registry-builder
```

Comment:
This was like 3 years old, so I bumped it, but if you think I should leave it on `v1.13.9` I will :smile: